### PR TITLE
Encapsulate and unify google_maps_key usage

### PR DESCRIPTION
**Pull request by elazarg**
Make sure there is only one place that gets this key, and standardize its name across the project. It's usages are now easily grepped.

BTW: is it okay to log the key? I thought it is a secret.

### CODE REVIEW

**Comment by atalyaalon** [comment]
@elazarg good catch. Not OK to log the key. It's secret - please delete this part of the log

**Comment by Mano3** [comment]
@elazarg I can't seem to find where the code logs the key. If it does, could you please remove that log?
I'll merge after this issue is resolved.

File: anyway/parsers/location_extraction.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def geocode_extract(location, maps_key):
         if road_no is None and extract_road_number(location) is not None:
             road_no = extract_road_number(location)
     except Exception as _:
-        logging.info('geocode extract location {0} maps key {1}'.format(location, maps_key))
+        logging.info('geocode extract location {0} maps key {1}'.format(location, google_maps_key))
```
Comment (elazarg) [inline anyway/parsers/location_extraction.py:167]: This is the place

**Comment by elazarg** [comment]
@Mano3 done.

**Pull request merged**